src/components/CommentBody/web/List.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { List, ListItem as CommentListItem} from '../../List'
+
+export const ListItem = ({ children }) => (
+  <CommentListItem style={{
+    fontSize: 'inherit',
```

Comment:
is this needed? Actual text is always in a child paragraph right?

Reply:
It's needed for the list style items (1., 2., 3.) which are :before items to match the font cut, otherwise we'd end up with this mismatch:
<img width="217" alt="screen shot 2018-04-30 at 17 35 57" src="https://user-images.githubusercontent.com/23520051/39436036-2e546440-4c9d-11e8-988a-43709a4a2426.png">